doc/PROGMEM.rst

```diff
@@ -0,0 +1,266 @@
+Guide to PROGMEM on ESP8266 and Arduino IDE
+===========================================
+
+Intro
+-----
+
+On low memory devices like the arduino and esp8266 you do not want
+strings to be stored in RAM. This occurs by default on these systems.
```

Comment:
slightly ambiguous sentence.  Suggest reworking it like...
"With Arduino the string constants are stored in RAM by default.  On low memory devices like Avr and Esp8266 you want to keep as much RAM available to your sketch as possible."